### PR TITLE
Feature/at 6846

### PR DESCRIPTION
**Pull request by robertmccardell-ccpo**

### CODE REVIEW

**Review by ericyoungquist-ccpo — CHANGES_REQUESTED** [review]

File: src/wizard/Step3/components/CreateApplicationForm.vue [hunk 1]
```diff
@@ -5,17 +5,17 @@
         Let’s create your new application
       </h1>
       <p v-if="!isStepTouched">
-        In this section, we will set up your cloud workspaces within
-        {{ this.$store.getters.getPortfolio.csp }}. If you have more than one
-        application, we will walk through them one at a time. Select 
+        {{ this.$store.getters["wizard/getPortfolio"].csp }}. If you have more
```
Comment (ericyoungquist-ccpo) [inline src/wizard/Step3/components/CreateApplicationForm.vue:8]: should probably not use the getter directly here - I know this is old code just refactored for the /wizard getter, but it should be a computed value that returns either the CSP from the wizard store, or the phrase "your selected CSP" if no CSP is set. also line 18 below.

File: src/wizard/ValidatableWizardStep.vue [hunk 2]
```diff
@@ -49,7 +55,10 @@ export default class ValidatableWizardStep<TModel> extends Validatable {
       if (this.skipValidation) return;
 
       this.incomingModel = JSON.parse(JSON.stringify(this.model)) as TModel;
-      this.touched = await this.$store.dispatch("isStepTouched", this.step);
+      this.touched = await this.$store.dispatch(
+        "wizard/isStepTouched",
```
Comment (ericyoungquist-ccpo) [inline src/wizard/ValidatableWizardStep.vue:59]: I know this was just a refactor for /wizard, but `isStepTouched` is a getter, not an action. - so it's always undefined, which throws an error and makes `this.touched` undefined, so doesn't validate on line 63.

File: src/wizard/Step5/components/ApplicationsEnvironmentsSummaryCard.vue [hunk 3]
```diff
@@ -2,7 +2,7 @@
   <div>
     <div v-for="application in applicationData" :key="application.id">
       <div class="review-table">
-        <v-card class="v-card ma-1 mb-10 body">
+        <v-card class="v-card ma-1 px-2 mb-10 body">
```
Comment (ericyoungquist-ccpo) [inline src/wizard/Step5/components/ApplicationsEnvironmentsSummaryCard.vue:5]: please remove `px-2`

File: src/wizard/Step5/components/ApplicationsEnvironmentsSummaryCard.vue [hunk 4]
```diff
@@ -29,7 +29,7 @@
               {{ application.description }}
             </p>
           </v-card-title>
-          <v-card-text class="body-lg text--base-darkest pa-6 pt-0">
+          <v-card-text class="body-lg text--base-darkest">
```
Comment (ericyoungquist-ccpo) [inline src/wizard/Step5/components/ApplicationsEnvironmentsSummaryCard.vue:32]: please add `pa-6 pt-0` back in

File: src/wizard/Step5/components/PortfolioSummary.vue [hunk 5]
```diff
@@ -82,11 +82,13 @@ export default class PortfolioSummary extends mixins(ApplicationData) {
...
-    ].filter((item) => this.$store.state.erroredSteps.indexOf(item.id) > -1);
+    ].filter(
+      (item) => this.$store.state.wizard.erroredSteps.indexOf(item.id) > -1
+    );
   }
 
   created(): void {
-    this.portfolio = this.$store.getters.getPortfolio;
+    this.portfolio = this.$store.getters["wizard/getPortfolio"];
     this.taskOrders = this.$store.getters.getTaskOrders;
```
Comment (ericyoungquist-ccpo) [inline src/wizard/Step5/components/PortfolioSummary.vue:92]: when `getTaskOrders` gets moved to wizard getters, don't forget about this guy

File: src/store/modules/wizard/actions.ts [hunk 6]
```diff
@@ -0,0 +1,458 @@
...
+    //store the applications
+    commit(
+      "applications/setCurrentApplications",
+      applicationData.applications,
+      { root: true }
+    );
+    commit("applications/initializeRootAdministrators", null, { root: true });
+
+    const rootAdmins = applicationData.operators.map((operator: Operator) => {
+      // const operatorModels: OperatorModel = {
```
[the remark this answers is not in the document]
Comment (robertmccardell-ccpo) [reply src/store/modules/wizard/actions.ts:412]: I think I see what's happening here...   I need to update this code slightly.

File: src/mixins/WizardModuleData.ts [hunk 7]
```diff
@@ -0,0 +1,27 @@
+import { Component } from "vue-property-decorator";
+import Vue from "vue";
+import { State, Action, Getter } from "vuex-class";
+import { WizardState, PortfolioSteps } from "@/store/modules/wizard/types";
+// import {
```
[the remark this answers is not in the document]
Comment (robertmccardell-ccpo) [reply src/mixins/WizardModuleData.ts:5]: nope... getting rid of it

File: src/store/modules/wizard/actions.ts (hunk 6 above)
[the remark this answers is not in the document]
Comment (robertmccardell-ccpo) [reply src/store/modules/wizard/actions.ts:412]: This should return the operatorModel

File: src/wizard/Step3/components/CreateApplicationForm.vue (hunk 1 above)
Comment (robertmccardell-ccpo) [reply src/wizard/Step3/components/CreateApplicationForm.vue:8]: fixed this with a computed

File: src/wizard/ValidatableWizardStep.vue (hunk 2 above)
Comment (robertmccardell-ccpo) [reply src/wizard/ValidatableWizardStep.vue]: fixed this... I caught it earlier.

File: src/wizard/Step5/components/ApplicationsEnvironmentsSummaryCard.vue (hunk 3 above)
Comment (robertmccardell-ccpo) [reply src/wizard/Step5/components/ApplicationsEnvironmentsSummaryCard.vue:5]: done..

File: src/wizard/Step5/components/ApplicationsEnvironmentsSummaryCard.vue (hunk 4 above)
Comment (robertmccardell-ccpo) [reply src/wizard/Step5/components/ApplicationsEnvironmentsSummaryCard.vue:32]: done

File: src/wizard/Step5/components/PortfolioSummary.vue (hunk 5 above)
Comment (robertmccardell-ccpo) [reply src/wizard/Step5/components/PortfolioSummary.vue]: yup..  Fixed this... I just referenced it from the task orders store...  Thanks

**Review by ericyoungquist-ccpo — APPROVED** [review]

**Review by ericyoungquist-ccpo — APPROVED** [review]

**Pull request merged**